### DISCUSSION TITLE
Chat Hangs When Adding Messages to Conversations with Deleted Models

### DISCUSSION THREAD

**Discussion by Fakamoto** [danny-avila/LibreChat#2073] [Feature Requests & Suggestions]
### What happened?

When attempting to add a new message to an old conversation that was previously held with a now-deleted model, the chat interface hangs indefinitely on loading and fails to send the new message. This issue persists until the user manually selects an existing model from the dropdown menu within the conversation. The expected behavior would be either a seamless continuation of the conversation using a default or previously selected model, or a prompt/notification that the model has been deleted and a new selection is necessary.

### Steps to Reproduce

1. Delete a model that has been used in conversations within the chat interface.
2. Navigate to an old conversation that was conducted using the now-deleted model.
3. Attempt to add a new message to the conversation.
4. Observe that the chat interface hangs on loading and does not send the message.

### Expected Behavior
Upon attempting to interact with a conversation tied to a deleted model, the chat should either:

* Automatically switch to a default or last used model that is still available, allowing the user to continue the conversation without interruption.
* Provide a clear notification or prompt that the model used in this conversation has been deleted, guiding the user to select a new model before continuing.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Not necessarily a bug since the app is doing exactly what it should be doing, blocking a non-recognized model. The UX for this can be improved though.
